framework/source/class/qx/util/DynamicScriptLoader.js

```diff
@@ -143,24 +143,36 @@ qx.Class.define("qx.util.DynamicScriptLoader", {
 
     /**
      * Start loading scripts. This may only be called once!
-     * @return {Promise} a promise which will be resolved after load of all scripts.
+     * @return {Promise|null} a promise which will be resolved after load of all scripts if promises are enabled or null if not.
```

Comment:
It appears this documentation is incorrect. I don't see anyplace that returns `null`; rather, line 167 simply does `return;` so the return value is `undefined`, not `null`. Either the documentation here should be corrected, or line 167 should be changed to `return null;`.